web/src/main/webapp/WEB-INF/jsp/include/showTableForEventDefinition.jsp

```diff
@@ -155,7 +157,7 @@
 				<tr>
 					<c:set var="isFirstLink" value="${true}" />
 					<c:if test="${table.filtered}">
-						<td class="table_tools"><a href="<c:out value="${removeFilterQuery}"/>">Clear Search Keywords</a></td>
+						<td class="table_tools"><a href="<c:out value="${removeFilterQuery}"/>"><fmt:message key="clear_search_keywords" bundle="${restext}"/></a></td>
 						<c:set var="isFirstLink" value="${false}" />
```

Comment:
I used the already defined key in the property file (notes.properties), and I think we totally could use this key, are we?